Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM node:8.11.4-slim
+FROM node:8.14.0-slim
 
 RUN apt-get update -y
-RUN apt-get install -y libpng12-dev
+# RUN apt-get install -y libpng12-dev
```

Comment:
@bloodbare any particular reason why we install libpng12-dev here? When I comment it out all tests still pass. If we want to keep that dependency we should use "RUN apt-get install -y libpng-dev" instead.